install_config/configuring_sdn.adoc

```diff
@@ -164,6 +164,147 @@ Check VNIDs by running:
+
+[NOTE]
+====
+Separate VSP Zone is created for each {product-title} project ensuring isolation amongst the projects.
+====
+
+[[operations-workflow]]
+
+==== Operations workflow
+
+This workflow is used by operations teams doing production roll outs of their applications.
+
+In this workflow, the network and security policies are first configured on the VSD in accordance with the rules set by the organization to deploy applications. Administrative users can potentially create multiple zones and subnets and map them to the same project using labels. While spinning up the pods, the user can use the Nuage Labels to specify what network a pod needs to attach to and what network policies need to be applied to it. This allows for sophisticated deployments where inter- and intra-project traffic can be controlled in a fine-grained manner. For example, inter-project communication is enabled on a project by project basis. This may be used to connect projects to common services that are deployed in a shared project.
+
+[[integration-components]]
+
+=== Installation
+
+The VSP integration with {product-title} works for both virtual machines (VMs) and baremetal installations of {product-title}.
+
+Ansible is the preferred method of installing {product-title} with Nuage VSP as the network plugin. Refer to the xref:index.adoc#install-config-index[{product-title} installation and configuration section] for details on how to install {product-title} using Ansible.
+
+==== Installation for single master
+
+In the Ansible nodes file, the following parameters need to be specified in order to setup Nuage VSP as the network plugin.
+
+====
+----
+ # Nuage specific parameters
+ openshift_use_openshift_sdn=False
+ openshift_use_nuage=True
+ os_sdn_network_plugin_name='nuage/vsp-openshift'
+ openshift_node_proxy_mode='userspace'
+ nuage_openshift_monitor_rest_server_port=9443
+
+ # VSP related parameters
+ vsd_api_url=https://192.168.103.200:8443
+ vsp_version=v4_0
+ enterprise=nuage
+ domain=openshift
+ vsc_active_ip=192.168.103.201
+ vsc_standby_ip=192.168.103.202
+ uplink_interface=eth0
+
+ # rpm locations
+ nuage_openshift_rpm=http://location_of_rpm_server/openshift/RPMS/x86_64/nuage-openshift-monitor-4.0.X.1830.el7.centos.x86_64.rpm
+ vrs_rpm=http://location_of_rpm_server/openshift/RPMS/x86_64/nuage-openvswitch-4.0.X.225.el7.x86_64.rpm
+ plugin_rpm=http://location_of_rpm_server/openshift/RPMS/x86_64/vsp-openshift-4.0.X1830.el7.centos.x86_64.rpm
+
+ # Optional parameters
+ nuage_interface_mtu=1460
+ nuage_master_adminusername=admin
+ nuage_master_adminuserpasswd=admin
+ nuage_master_cspadminpasswd=admin
+ nuage_openshift_monitor_log_dir=/var/log/nuage-openshift-monitor
+
+ # Required for brownfield install (where a {product-title} cluster exists without Nuage as the networking plugin)
```

Comment:
the {product-title} macro is not expanding when in a code snippet tag. Can we just change this to Openshift?